src/pages/Index.vue

```diff
@@ -2,18 +2,120 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    
+    <header class="header">
+    </header>
+      <body>
```

Comment:
header and body are already defined in the layout. Can you remove them? also the `</body>` later in the page

Reply:
ok, great..I'll take them out :)